
base_model: 'yolov8n.pt'
custom_model: '/home/praveen/alfred-autonomy/src/perception/object_detection/model/sodacan_tissue.pt'

# subscribe_topic: '/camera/color/small_image'
subscribe_topic: '/camera/color/image_raw'

publish_topic: 'object_bounding_boxes'
visualize: true
visualize_topic: 'annotated_image_body'

device_1: 'cuda:0'
device_2: 'cuda:1'

class_list:
  - 'soda_can'
  - 'tissue_paper'
  - 'toothbrush'
  - 'tie'
  - 'cell phone'
  - 'banana'
  - 'apple'
  - 'orange'
  - 'bottle'
  - 'cup'
  - 'teddy_bear'



detic:
  annotation_topic: /camera/color/detic_annotated
  config_path: /usr/local/lib/obj_det_env/Detic/configs/detectron/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.yaml
  device: 'cuda:2'
  weight_path: 'https://dl.fbaipublicfiles.com/detic/Detic_LCOCOI21k_CLIP_SwinB_896b32_4x_ft4x_max-size.pth'
  
  vocab_name : 'lvis'
  
  custom_vocab: true
  class_list: [
      "toothbrush", "toothpaste",
      "keys", 
      "remote control", "cell phone", "watch",
      "apple", "orange"
  ]